index.js

```diff
@@ -6,15 +6,29 @@ module.exports = {
   name: require('./package').name,
 
   setupPreprocessorRegistry(type, registry) {
+    if (type !== 'parent') {
+			return;
+		}
     let checker = new VersionChecker(this.project);
     let dep = checker.for('ember-source');
-    let maybePlugin = {
+    let version = dep.gte('3.17.0');
+		const plugin = this._buildPlugin({ version });
```

Comment:
Nitpick: It seems there is a combination of tabs and spaces present in this file which I would make consistent. (it's only visible if you configure GitHub to show tabs as something else than 2 spaces). 

This is what it looks like with tabs displayed as 4 spaces:
![image](https://user-images.githubusercontent.com/3533236/188480000-31bcd9c0-340c-4b4b-8979-29090038e620.png)